x-pack/plugins/observability_solution/observability_onboarding/public/application/packages_list/index.tsx

```diff
@@ -78,7 +78,8 @@ const PackageListGridWrapper = ({
   if (!isInitialHidden && isLoading) return <Loading />;
 
   const showPackageList =
-    (showSearchBar && !isInitialHidden) || showSearchBar === false;
+    (showSearchBar && !isInitialHidden && !!searchQuery) ||
```

Comment:
We might not need the `isInitialHidden` anymore as `searchQuery` will be empty when page loads and will prevent the list from showing up. But I might be missing something.

Reply:
I think you are right, let's delete some code 😃